### PR TITLE
LPS-174982 Replace aui:icon with clay:icon in product-navigation-control-menu-web

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://issues.liferay.com/browse/LPS-174982)

## Test Steps

1.  Open a Widget Page
2. See Control Menu bar up to the right 
Screenshot 2023-02-07 at 18 08 28

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_body.jsp [hunk 1]
```diff
@@ -32,7 +32,11 @@
 				<span class="font-weight-bold"><liferay-ui:message key="add" /></span>
 
 				<a aria-label="<%= LanguageUtil.get(request, "close") %>" class="sidenav-close text-secondary" href="javascript:void(0);">
```
Comment (ealonso) [inline modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_body.jsp:34]: @sandrodw3 @DiegoHu97 Why not a clay:link?

File: modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_icon.jsp [hunk 2]
```diff
@@ -46,7 +46,11 @@ String portletNamespace = PortalUtil.getPortletNamespace(ProductNavigationContro
 		href="javascript:void(0);"
 		id="<%= portletNamespace %>addToggleId"
 	>
-		<aui:icon cssClass="icon-monospaced" image="plus" markupView="lexicon" />
```
Comment (ealonso) [inline modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_icon.jsp:49]: @DiegoHu97 @sandrodw3 same here

Also if we cannot use a clay:link, Why is necessary span tag?

File: modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_body.jsp (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_body.jsp:34]: Agree, this should be better be a `clay:link` 😛

File: modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_icon.jsp (hunk 2 above)
Comment (sandrodw3) [reply modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_icon.jsp:49]: Agree as well

Comment (sandrodw3) [reply modules/apps/product-navigation/product-navigation-control-menu-web/src/main/resources/META-INF/resources/entries/add_content_icon.jsp:49]: I'll help you with both @DiegoHu97

**Comment by sandrodw3** [comment]
ci:forward

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/11087

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**